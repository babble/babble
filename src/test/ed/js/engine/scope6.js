
option = { a : "1" };
opt = 1;

function(t) {
    for( opt in option ) {
        print( opt );
    }
};

print( opt );
