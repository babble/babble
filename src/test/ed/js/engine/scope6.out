1
7
10
7
